packages/elements/src/components/ino-segment-button/readme.md

```diff
@@ -67,18 +67,6 @@ class MyComponent extends Component {
 | `checkedChange` | Emits if the user interacts with the button. If the button is disabled or checked, the event will not be emitted. | `CustomEvent<any>` |
 
 
-## CSS Custom Properties
```

Comment:
This probably did not happen on purpose.

Reply:
Nope - this happened by accident. Should be fixed now.